source/controller/project_controller.py

```diff
@@ -429,14 +431,15 @@ def on_save_graph(self):
 
         file_path = project_information_store.file_path
         file_name, file_type = os.path.splitext(file_path)
+        current_graph = project_information_store.current_graph
         graph = None
         new_g6 = ""
         replaced_line = ""
 
-        if self.edited_graph is None:
+        if current_graph is None:
             return
         else:
-            new_g6 = nx.to_graph6_bytes(self.edited_graph)[10:-1].decode("utf-8")
+            new_g6 = nx.to_graph6_bytes(current_graph)[10:-1].decode("utf-8")
```

Comment:
I looked up the use of this [10,-1], so I understand that you want to remove the word "graph6". Looking at the [networkx documentation](https://networkx.org/documentation/stable/reference/readwrite/generated/networkx.readwrite.graph6.to_graph6_bytes.html#networkx.readwrite.graph6.to_graph6_bytes) I saw that there is a function for that. So I ask you to adapt this function using this feature.

`nx.to_graph6_bytes(current_graph,header=False).decode("utf-8")`

Reply:
It was also necessary to insert [:-1] to remove the\n from the g6 code